fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -233,6 +234,22 @@ Key getLogKey(Version version, Key backupUid) {
 	return k2.withPrefix(backupLogKeys.begin);
 }
 
+namespace {
```

Comment:
do we want to test cases when `blockSize` is not default?

Reply:
Added a different `blockSize` for tests. In the code, we always use the default value.